fusor-ember-cli/app/routes/openstack/overcloud.js

```diff
@@ -28,25 +28,30 @@ export default Ember.Route.extend({
   },
 
   deactivate() {
-    this.updateNeutronPublicInterface();
-    this.send('updatePlanParameters', [{
-      name: 'Controller-1::NeutronPublicInterface',
-      value: this.get('controller').get('neutronPublicInterface')
-    }]);
+    var controller = this.get('controller'),
+      changedParams = [
+        {
+          name: 'Controller-1::NeutronPublicInterface',
+          value: this.get('controller').get('neutronPublicInterface')
+        },
+        {
+          name: 'Controller-1::AdminPassword',
+          value: this.get('controller').get('overcloudPassword')
+        }
+      ];
+
+    this.updateLocalPlanParameters(changedParams);
+    this.send('updatePlanParameters', changedParams);
     return this.send('saveDeployment', null);
   },
```

Comment:
So I've followed the pattern saving on deactivate, but I'm unsure of doing it this way in general.  If we have an invalid form, and we close the browser or navigate away, doesn't that trigger the save?